src/call.c

```diff
@@ -869,7 +871,7 @@ int call_alloc(struct call **callp, const struct config *cfg, struct list *lst,
 	       struct account *acc, struct ua *ua, const struct call_prm *prm,
 	       const struct sip_msg *msg, struct call *xcall,
 	       struct dnsc *dnsc,
-	       call_event_h *eh, void *arg)
+	       call_event_h *eh, void *arg, const char *user_data)
```

Comment:
Is it possible to use a setter function instead of extending the parameter list?

Reply:
OK, I've added a setter.